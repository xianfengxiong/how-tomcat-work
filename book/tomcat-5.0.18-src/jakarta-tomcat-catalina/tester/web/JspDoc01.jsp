<?xml version="1.0"?>
<jsp:root xmlns:jsp="http://java.sun.com/JSP/Page" version="1.2">
  <a>text<b></b></a>
  <jsp:text>&lt;a&gt;text&lt;b&gt;&lt;/b&gt;&lt;/a&gt;</jsp:text>
  <c><d>text</d></c>
  <jsp:text>&lt;c&gt;&lt;d&gt;text&lt;/d&gt;&lt;/c&gt;</jsp:text>
  <e><f></f>text<f></f></e>
  <jsp:text>&lt;e&gt;&lt;f&gt;&lt;/f&gt;text&lt;f&gt;&lt;/f&gt;&lt;/e&gt;</jsp:text>
</jsp:root>
