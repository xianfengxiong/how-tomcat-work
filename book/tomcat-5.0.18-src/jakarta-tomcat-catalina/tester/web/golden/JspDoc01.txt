<a>text<b/></a><a>text<b></b></a><c><d>text</d></c><c><d>text</d></c><e><f/>text<f/></e><e><f></f>text<f></f></e>
